_sass/_layout.scss

```diff
@@ -295,11 +334,90 @@ footer {
 }
 
 // tablet
-@media only screen and (max-width: 1000px){
+@media only screen and (max-width: 700px){
+  .navigation {
+    background-color: #242424;
+    height: 100%; /* Specify a height */
+    width: 0; /* 0 width - change this with JavaScript */
+    position: fixed; /* Stay in place */
+    z-index: 2; /* Stay on top */
+    top: 0;
+    left: 0;
+    overflow-x: hidden; /* Disable horizontal scroll */
+    padding-top: 60px; /* Place content 60px from the top */
+    transition: 0.35s; /* 0.5 second transition effect to slide in the sidepanel */

```

Comment:
Right now, at the 700px breakpoint the behavior of the entire `nav-bar` element changes from being fixed at the top of the screen, to expanding from the side when the menu button is clicked/tapped. Although this transition is only seen when resizing a browser window, it's a bit jarring since the _entire_ `nav-bar` element disappears, being replaced by the `page-header` element which was previously hidden underneath. 

Instead, I would suggest having just the `topBarLinks` element disappear once there isn't enough space in `nav-bar` for them, leaving the logo and a new sidebar button in `nav-bar` at the top. This is how most sites handle this type of transition (ex. our favorites uber.com and lyft.com)